partials/_general.scss

```diff
@@ -0,0 +1,203 @@
+body {
+    margin: 0;
+}
+
+.header {
+    height: 48px;
+    font-weight: 400;
+    font-size: $nav-font-size;
+
+    @include media ( ">=md" ) {
+        height: 56px;
+    }
+
+    @include media ( ">=lg" ) {
+        height: 125px;
+    }
+
+    &__container {
+        max-width: 1264px;
+        margin: 0 auto;
+        width: 100%;
+    }
+}
+
+.hamburger {
+    position: absolute;
+    top: 0;
+    right: 0;
+    margin: 0;
+
+    @include media ( ">=lg" ) {
+        display: none;
+    }
+
+    &__img {
+        margin: 16px 16px 0px;
+        cursor: pointer;
+    
+        @include media ( ">=md" ) {
+            margin: 19px 16px 0px;
+        }
+    }
+}
+
+.nav {
+    line-height: 20px;
+    color: $tws_black;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    gap: 10px;
+    width: 100%;
+    position: relative;
+    height: 48px;
```

Comment:
<img width="348" alt="Zrzut ekranu 2022-12-1 o 14 34 59" src="https://user-images.githubusercontent.com/87694556/205067977-6e835170-100d-41fe-b606-f39c7e7e545b.png">
your nav is 48px while the one in figma is 56px

Reply:
I think the S screen size is 48px unless I'm mistaken? Then increases to 56px from M updwards?